packages/core/src/lib/TwaGenerator.ts

```diff
@@ -69,17 +70,20 @@ const DELETE_FILE_LIST = [
   'app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml',
 ];
 
+const SPLASH_IMAGES: IconDefinition[] = [
+  {dest: 'app/src/main/res/drawable-hdpi/splash.png', size: 450},
```

Comment:
currently these are sorted by |dest| alphabetically. IMHO, it would make more sense to sort by |size|.

Reply:
Good point. Changed.